### DISCUSSION TITLE
how to recover a branch deleted a year ago? reflog just set 180 days expires

### DISCUSSION THREAD

**Discussion by pengcheng-zh** [community/community#37182] [Other Feature Feedback, Questions, & Ideas]
I delete the branch a year ago, and i can't get any log of the branch, because the reflog expire config default to 180

**Comment by radar-blip** [comment]
I'm not so sure you can if it was that long ago. I remember reading something about a 90 limit.
You can always check though. Maybe here,  under restore deleted data [Support Page](https://support.github.com/request)

**Comment by ossanna16** [comment]
Hi there @pengcheng-zh and welcome to our community! Thank you for asking a great question 🙂
